starter_code/src/movies.js

```diff
@@ -1,20 +1,101 @@
 /* eslint no-restricted-globals: 'off' */
-// Turn duration of the movies from hours to minutes 
+// Turn duration of the movies from hours to minutes
 
+function turnHoursToMinutes(a) {
+  var out = a.map(function(e) {
+    var newElement = Object.assign({}, e)
```

Comment:
mmmm, me suena que yo os puse la declaración de la variable fuera del map... qué hace dentro otra vez???!!!! de esta forma estáis creando y asignando cada vuelta del map, de la otra solo asignando, lo que se traduce en mayor velocidad de ejecución